documentation/src/main/asciidoc/reference/mapping-indexedembedded.asciidoc

```diff
@@ -552,4 +599,4 @@ This mapping will declare the following fields in the `Book` index:
 ----
 include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/indexedembedded/onelevel/IndexedEmbeddedOneLevelIT.java[tags=programmatic]
 ----
-====
\ No newline at end of file
+====
```

Comment:
yeah, I think it comes from IDE (Settings -> Editor -> General -> Ensure line feed at file end on Save) and I might have it in git as well (an old habit I guess 🙈🥲)
I may or may not be wrong, but I think that the Eclipse formatter does this too, (or maybe I've added that in the config for it) and my idea was to have these blank lines added while we are modifying the files anyway so that when we do the mass formatting there'll be fewer changes...

Reply:
> my idea was to have these blank lines added while we are modifying the files anyway so that when we do the mass formatting there'll be fewer changes...

Good point